<ui version="4.0" >
 <class>CreateNewDialog</class>
 <widget class="QDialog" name="CreateNewDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create new ...</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="4" colspan="2" >
    <widget class="QCheckBox" name="cbAddToProject" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string>&amp;Add to project</string>
     </property>
     <property name="icon" >
      <iconset resource="../images/Edyuk.qrc" >:/add.png</iconset>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5" >
    <widget class="QToolButton" name="tbLocation" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset/>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLabel" name="lblName" >
     <property name="text" >
      <string>Name</string>
     </property>
     <property name="buddy" >
      <cstring>leName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="cbFilter" />
   </item>
   <item row="2" column="2" >
    <widget class="QCheckBox" name="cbOpen" >
     <property name="text" >
      <string>&amp;Open</string>
     </property>
     <property name="icon" >
      <iconset resource="../images/Edyuk.qrc" >:/open.png</iconset>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="1" column="0" colspan="2" >
    <widget class="QListWidget" name="lwTemplates" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>13</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy" >
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="iconSize" >
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="movement" >
      <enum>QListView::Static</enum>
     </property>
     <property name="flow" >
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="isWrapping" stdset="0" >
      <bool>false</bool>
     </property>
     <property name="viewMode" >
      <enum>QListView::ListMode</enum>
     </property>
     <property name="uniformItemSizes" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLabel" name="lblLocation" >
     <property name="text" >
      <string>Location</string>
     </property>
     <property name="buddy" >
      <cstring>leLocation</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Category filter </string>
     </property>
     <property name="buddy" >
      <cstring>cbFilter</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3" colspan="2" >
    <widget class="QLineEdit" name="leLocation" >
     <property name="toolTip" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="3" >
    <widget class="QLineEdit" name="leName" >
     <property name="toolTip" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="4" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="0" colspan="2" >
       <widget class="QTextEdit" name="eDescription" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>4</hsizetype>
          <vsizetype>4</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="horizontalScrollBarPolicy" >
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="undoRedoEnabled" >
         <bool>false</bool>
        </property>
        <property name="lineWrapMode" >
         <enum>QTextEdit::WidgetWidth</enum>
        </property>
        <property name="readOnly" >
         <bool>true</bool>
        </property>
        <property name="acceptRichText" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Category</string>
        </property>
        <property name="buddy" >
         <cstring>leType</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="leType" >
        <property name="readOnly" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="6" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbFilter</tabstop>
  <tabstop>lwTemplates</tabstop>
  <tabstop>leName</tabstop>
  <tabstop>leLocation</tabstop>
  <tabstop>tbLocation</tabstop>
  <tabstop>cbOpen</tabstop>
  <tabstop>cbAddToProject</tabstop>
  <tabstop>leType</tabstop>
  <tabstop>eDescription</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../images/Edyuk.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateNewDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>428</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel" >
     <x>370</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateNewDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>532</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel" >
     <x>562</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
